airflow/ui/src/utils/tokenHandler.ts

```diff
@@ -23,7 +23,7 @@ const getTokenFromCookies = (): string | undefined => {
   const cookies = document.cookie.split(";");
 
   for (const cookie of cookies) {
-    if (cookie.startsWith("_token=")) {
+    if (cookie.includes("_token=")) {
       const [, token] = cookie.split("=");
```

Comment:
I'm not sure this is right anymore. You could end up with cookie having a value of

```
something_else=value:_token=token
```
So token would end up as `value:_token=token`

Reply:
Updated for something ideally more robust